src/share/vm/prims/unsafe.cpp

```diff
@@ -1629,33 +1684,33 @@ static JNINativeMethod methods_15[] = {
     DECLARE_GETSETNATIVE(Float, F),
     DECLARE_GETSETNATIVE(Double, D),
 
-    {CC "getAddress",         CC "(" ADR ")" ADR,             FN_PTR(Unsafe_GetNativeAddress)},
```

Comment:
please don't change the unnecessary part

Reply:
Done.